### PR TITLE
Update endpoint tags

### PR DESCRIPTION
**Pull request by akshaaatt**
Replace the tagger endpoint with https://labs.api.listenbrainz.org/mbid-mapping/

### CODE REVIEW

File: app/src/main/java/org/metabrainz/android/data/sources/api/entities/Recording.kt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+package org.metabrainz.android.data.sources.api.entities
+
+class Recording : ArrayList<RecordingItem>()
```
Comment (amCap1712) [inline app/src/main/java/org/metabrainz/android/data/sources/api/entities/Recording.kt:3]: Is this still needed?

**Review by amCap1712 — APPROVED** [review]
Changes look good but didn't test locally.

**Comment by amCap1712** [comment]
We also have an endpoint in LB which is used by the metadata viewer. It does the same lookup as this endpoint but also returns the additional metadata with it. So instead of making an API call to match release after the labs api endpoint, you could just make 1 api call. Maybe another thing to look into in future.

File: app/src/main/java/org/metabrainz/android/data/sources/api/entities/Recording.kt (hunk 1 above)
Comment (akshaaatt) [reply app/src/main/java/org/metabrainz/android/data/sources/api/entities/Recording.kt]: Makes sense. Removed.

**Comment by akshaaatt** [comment]
> We also have an endpoint in LB which is used by the metadata viewer. It does the same lookup as this endpoint but also returns the additional metadata with it. So instead of making an API call to match release after the labs api endpoint, you could just make 1 api call. Maybe another thing to look into in future.

This would be great! Let me know the endpoint and we'll make the required updates!

**Pull request merged**